### DISCUSSION TITLE
No support for database files in S3

### DISCUSSION THREAD

**Discussion by arnoldros** [xerial/sqlite-jdbc#778] [General]
Is there currently support for database files located in AWS S3?

s3path = "s3://my-bucket/myfile.db"
df = spark.read.format('jdbc').option('driver', 'org.sqlite.JDBC').option('url', f"jdbc:sqlite:{s3path}").option('dbtable', 'mytable').load()

Error:  
java.sql.SQLException: path to 's3://my-bucket/myfile.db': '/home/app/s3:' does not exist

**Comment by gotson** [comment]
There is not. You should consider mounting the s3 bucket on your filesystem which would be transparent to java and the native C library.
